### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by kannon92**
Getting some pieces working so I can deploy to kind.

### CODE REVIEW

File: config/samples/deploy_armada.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ spec:
   ingress:
     ingressClass: "nginx"
   clusterIssuer: "k8s-cluster-issuer"
-  hostNames: [localhost]
+  hostNames: [localhost-server]
```
Comment (kannon92) [inline config/samples/deploy_armada.yaml:10]: Makes sense but these hostnames need to be unique. These are meaningless right now but we should make sure each service has a unique ip address.

File: dev/helm-charts/redis_bitnami_values.yaml [hunk 2]
```diff
@@ -1,2 +1,5 @@
 auth:
   enabled: false
+
+replica:
```
Comment (kannon92) [inline dev/helm-charts/redis_bitnami_values.yaml:4]: Before this change redis would start 3 replicas.

File: controllers/install/lookout_controller.go [hunk 3]
```diff
@@ -49,6 +49,7 @@ type LookoutReconciler struct {
 
 //+kubebuilder:rbac:groups=install.armadaproject.io,resources=lookouts,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=install.armadaproject.io,resources=lookouts/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups="batch",resources=jobs,verbs=get;list;watch;create;delete;deletecollection;patch;update
```
Comment (ClifHouck) [inline controllers/install/lookout_controller.go:52]: What caused this change?

Comment (kannon92) [reply controllers/install/lookout_controller.go:52]: Now that we are starting jobs by default, the rbac was complaining that it didn't have permission to actually create the batch jobs.  So I added it here so now we are able to actually create those jobs.  

You can find this by looking in the logs of an operator that is deployed to Kind.

**Review by ClifHouck — APPROVED** [review]

**Pull request merged**